### ISSUE TITLE
Downtime untill Jul26

### ISSUE THREAD

**Issue by JoaoLaboissiere2013** [PenguinMod/PenguinMod-Home#120]
When is it comming to a end

**Comment by RedMan13** [comment]
if you keep asking

**Comment by RedMan13** [comment]
:trollface:

**Comment by dumorando** [comment]
i jusst downloaded the repo and ran it witht  he public url without donating is this allowed

**Comment by PuzzlingGGG** [comment]
> i jusst downloaded the repo and ran it witht he public url without donating is this allowed

yes because its open source but you wont get our db

**Comment by dumorando** [comment]
i meant like if i was allowed to make an account
![image](https://github.com/user-attachments/assets/beec31e9-cef9-4b56-b472-2aebdbbf20b7)

**Comment by RedMan13** [comment]
uhm

**Comment by JeremyGamer13** [comment]
just dont make a url publicly accessible and its fine

**Comment by polygoners1** [comment]
> i meant like if i was allowed to make an account ![image](https://private-user-images.githubusercontent.com/144301348/352268252-beec31e9-cef9-4b56-b472-2aebdbbf20b7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ2ODUzODksIm5iZiI6MTcyNDY4NTA4OSwicGF0aCI6Ii8xNDQzMDEzNDgvMzUyMjY4MjUyLWJlZWMzMWU5LWNlZjktNGI1Ni1iNDcyLTJhZWJkYmJmMjBiNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNlQxNTExMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMzBiOGQ0OWY2OThmMjQwZWJkNDYwYTI2NGQzOWIyYjY0ODA0NDlmZDUyMDk4NTE4OGJiOTlhYWFjM2YzNmUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.LtXSNFuzsYdjhGPd_EvnUijf_WcM6ZtCw237j2Ub6I0)

btw the guy who made those was called "ianyour"gyatt"" and i saw those on the beta servers that you could accsess on the "deployments" tab for this repo. even before the beta server accsess on the discord.

**Comment by RedMan13** [comment]
yeah it wasn't supposed to be secret, just held t a group so we could control what on earth happenes without people sending 30 iterations of the same bug 🥱
